app/models/Organization.ts

```diff
@@ -74,3 +83,74 @@ export const getResourceLocations = (org: Organization): LocationDetails[] => {
     recurringSchedule: org.recurringSchedule,
   }));
 };
+
+// TODO Can we remove this?
+// const verifyItem = (item: Organization|Service, itemType: string) => {
+//   const { id } = item;
+//   const changeRequest = { verified_at: new Date().toISOString() };
+//   return post(`/api/${itemType}s/${id}/change_requests`, { change_request: changeRequest })
+//     .then(response => {
+//       // TODO: Do not use alert() for user notifications.
+//       if (response.ok) {
+//         alert(`${itemType.charAt(0).toUpperCase() + itemType.slice(1)} verified. Thanks!`);
+//       } else {
+//         alert(`Issue verifying ${itemType}. Please try again.`);
+//       }
+//     });
+// };
+
+export const getResourceActions = (
+  resource: Organization,
+  service?: Service,
+  filterActions?: string[],
+) => {
+  const phoneNumber = _.get(resource, 'phones[0].number');
+  const latitude = _.get(resource, 'addresses[0].latitude');
+  const longitude = _.get(resource, 'addresses[0].longitude');
+
+  // TODO Can we remove these unused items?
+  const actions: OrganizationAction[] = [
+    // {
+    //   name: 'Edit',
+    //   icon: 'edit',
+    //   to: `/organizations/${resource.id}/edit`,
+    // },
+    {
+      name: 'Print',
+      icon: 'print',
+      handler: () => { window.print(); },
+    },
+    // {
+    //   name: 'Mark Correct',
+    //   icon: 'verify',
+    //   handler: () => (service
+    //      ? verifyItem(service, 'service')
+    //      : verifyItem(resource, 'resource')
+    //   ),
+    // },
+    {
+      name: 'Share Feedback',
+      icon: 'feedback',
+    },
+  ];
+
+  if (phoneNumber) {
+    actions.push({
+      name: 'Call',
+      icon: 'phone',
+      link: `tel:${phoneNumber}`,
+    });
+  }
+
+  if (latitude && longitude) {
+    actions.push({
+      name: 'Directions',
+      icon: 'directions',
+      link: `http://google.com/maps/dir/?api=1&destination=${latitude},${longitude}`,
+    });
+  }
+
+  return filterActions
+    ? actions.filter(a => filterActions.indexOf(a.icon) > -1)
```

Comment:
```suggestion
    ? actions.filter(a => filterActions.includes(a.icon))
```

I think `Array.prototype.includes()` is probably a clearer method to use here.